libs/shared/ui-elements/src/lib/prio-button/prio-button.component.ts

```diff
@@ -9,10 +11,20 @@ import { Component, OnInit, Input, Output, EventEmitter, HostBinding } from '@an
 
 export class PrioButtonComponent implements OnInit {
 
-  inputFocus = false;
+  @Input() style = 'solid' ;  //solid || outline || flat
+  @Input() type = 'btn-primary' ;  //btn-primary || btn-secondary || btn-danger, ...etc
```

Comment:
I think this property is related to colours and styling so that should be named something else like swapping it with the style above and also prefixing them is better like 
`@Input() btnStyle:ButtonStyle = 'bt-primary' `
`@Input() btnType:ButtonType = 'solid' `

Reply:
done
although clarity is very confusing in this
in button section, 
in the design guidelines at the very beginning of page: it says three different types, flat, outline, solid

however in the code examples part it says:

Code & Examples
Clarity defines three button styles:

Solid. 
Outline. 
Flat.